bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/AsterismTypeConverters.scala

```diff
@@ -29,5 +29,8 @@ object AsterismTypeConverters {
       (GhostHigh, GhostHighResolutionTargetPlusSky, GhostStandard) -> GhostTargetPlusSky,
       (GhostHigh, GhostHighResolutionTargetPlusSky, GhostPRV) -> GhostHighResolutionTarget,
 
-      (GhostPRV, GhostHighResolutionTarget, GhostStandard) -> GhostSingleTarget)
+      (GhostPRV, GhostHighResolutionTarget, GhostStandard) -> GhostSingleTarget,
+
+      // This oddly happens sometimes?
```

Comment:
Why is this odd?  The comment doesn't help much.

Reply:
I think this was the result of a bug that I fixed. Let me try removing it and testing, but that combination of parameters should never occur. (I had an earlier incorrect entry in the table that was causing havoc but I found and repaired, and now everything in the conversions looks correct.)